Code/Framework/AzCore/AzCore/Memory/IAllocator.h

```diff
@@ -100,56 +97,19 @@ namespace AZ
     /**
      * Interface class for all allocators.
      */
-    class IAllocator
+    class IAllocator : public IAllocatorSchema
     {
     public:
-        IAllocator(IAllocatorAllocate* allocationSource);
+        IAllocator(IAllocatorSchema* schema = nullptr);
```

Comment:
Is this constructor still needed since IAllocator now inherits from IAllocatorSchema?

Any class that implements IAllocator would by definition implement IAllocatorSchema

Reply:
Nevermind: It looks like it is needed, for the cases of using something like the SystemAllocator with the MallocSchema